layers/+spacemacs/spacemacs-evil/packages.el

```diff
@@ -90,7 +90,10 @@
 
 (defun spacemacs-evil/init-evil-mc ()
   (use-package evil-mc
-    :defer t))
+    :defer t
+    :init
+    (setq mc/list-file (concat spacemacs-cache-directory "mc-lists.el"))
+    ))
```

Comment:
My bad, my personal style puts closing parens on their own line for cleaner diffs :)